Tools/CISupport/ews-build/steps.py

```diff
@@ -4957,20 +4961,25 @@ def evaluateCommand(self, cmd):
         rc = super(ValidateCommitMessage, self).evaluateCommand(cmd)
         if rc == SKIPPED:
             return rc
-        if rc != SUCCESS:
-            self.summary = 'Error parsing commit message'
-            return rc
 
-        log_text = self.log_observer.getStdout()
-        if '(OOPS!)' in log_text:
-            self.summary = 'Commit message contains (OOPS!)'
-            return FAILURE
-        if 'Reviewed by' not in log_text and 'Unreviewed' not in log_text:
-            self.summary = 'No reviewer information in commit message'
-            return FAILURE
+        if rc == SUCCESS:
+            self.summary = 'Validated commit message'
+            log_text = self.log_observer.getStdout()
+            if '(OOPS!)' in log_text:
```

Comment:
Do you want to do regex matching here for cases like OPPS like you did earlier?

Reply:
Good idea, will do that